apps/src/aichat/views/modelCustomization/compare-models-dialog.module.scss

```diff
@@ -14,7 +14,7 @@
 }
 
 div.modelComparisonDialog {
-  height: calc(100vh - 100px);
+  height: calc(90vh - 100px);
```

Comment:
Unrelated follow-up to https://github.com/code-dot-org/code-dot-org/pull/57775, turns out that at full screen this was overflowing on the bottom. Looks great at smaller heights (and now at full height!)